docs/features/java.rst

```diff
@@ -299,7 +299,7 @@ for both arrays and direct buffers.
 Supported APIs
 --------------
 
-Complete MPI-3.1 coverage is provided in the Open MPI Java bindings,
+Complete MPI-|mpi_standard_version|.|mpi_standard_subversion| coverage is provided in the Open MPI Java bindings,
```

Comment:
I think you want to leave this as hard-coded to MPI-3.1 -- we haven't ever extended the Java MPI APIs beyond MPI-3.1.